SPECS/lcov/lcov.spec

```diff
@@ -0,0 +1,144 @@
+Name: lcov
```

Comment:
Don't forget about updating the `licenses.json`, regenerating the licenses table, and updating `cgmanifest,json`.

To regenerate the licenses table:

```bash
python3 toolkit/scripts/licenses/license_map.py --only_update \
          SPECS/LICENSES-AND-NOTICES/data/licenses.json \
          SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md \
          SPECS \
          SPECS-SIGNED
```